src/components/Rocket.jsx

```diff
@@ -0,0 +1,48 @@
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
- [ ] Kindly fix the following behavior:
![chrome-capture-2023-8-8](https://github.com/wacoo/orbit-space-travel-agency/assets/24703171/549bc0cf-3d9c-4510-aefe-aa89201bf81a)

When I register for a mission or rocket and go to the profile page everything works fine, But after that, I visit the mission or rockets page and come back to the profile page. I don't find any registered missions or rockets.